.eslintrc.json

```diff
@@ -59,8 +59,25 @@
 		"comma-dangle": "error",
 		"keyword-spacing": "error",
 		"space-in-parens": ["error", "never"],
-		"func-style" : ["warn", "expression"],
+		"func-style" : ["error", "expression"],
 		"require-await": "off",
-		"@typescript-eslint/no-floating-promises": ["error"]
-	}
+		"@typescript-eslint/no-floating-promises": ["error"],
+		"jest/expect-expect": [
+			"error",
+			{
+				"assertFunctionNames": [
+					"expect",
+					"supertest.**.expect",
+					"*.expect"
```

Comment:
This is for making `someInternaMethod(xxx).expect(200)` to pass. We'll trust the author as long as they wrote `expect` somewhere.